### ISSUE TITLE
Test set performance

### ISSUE THREAD

**Issue by ffffffff0** [Allen517/cyanrnn_project#2]
Hello, I am confused as to how to validate the performance of the model on the test set

**Comment by Allen517** [comment]
Every behavior is composed by a tuple (user id, time). Thus, for next
activated user prediction, I use MRR in the paper. Meanwhile, the predicted
time is evaluated by RMSE.

**Comment by ffffffff0** [comment]
Thanks for your reply. How do I get the evaluation metrics for the test set when the model training in the code uses the validation set and the training set?
